packages/fxa-admin-panel/public/index.html

```diff
@@ -11,7 +11,6 @@
       name="viewport"
       content="width=device-width,initial-scale=1,minimum-scale=1,maximum-scale=2,user-scalable=yes"
     />
-    <link rel="stylesheet" href="styles/main.css" />
```

Comment:
@LZoog It occurred to me to see what this stylesheet was doing, and.. I don't think it does anything? In development it's not used because all the styles are rendered in inline style tags, and in a production build the chunked version gets appended to the head. Do you know if we actually need this?

Reply:
Huh... I think you're right. We don't have it in payments. TIL